### PR TITLE
update basic documentation

### PR DESCRIPTION
**Pull request by MrEconomical**
sync repl.it basic docs with basic github docs

### CODE REVIEW

File: misc/basic.md [hunk 1]
```diff
@@ -237,33 +214,6 @@ Example:
...
-
-#### CLC
-
-`CLC` clears the display.
-
-Example:
-```
-10 CLC
-```
-
```
Comment (amasad) [inline misc/basic.md:266]: why delet

Comment (MrEconomical) [reply misc/basic.md]: my bad, they were missing from the github basic documentation
https://github.com/amasad/pg-basic/pull/9

**Comment by amasad** [comment]
@MrEconomical why don't you document the `TIME` function?

**Comment by MrEconomical** [comment]
added, cool

**Pull request merged**